sysadmin-docs/multi-tenant.md

```diff
@@ -0,0 +1,92 @@
+# Multi-tenancy 
+
+Multi-tenancy is built-in uwazi and allows for multiple databases/tenants to be served with a single node process.
+the tenant object defines all specific configurations, tenants should be created on a separated db 'uwazi_shared_db' in a 'tenants' collections with the following structure:
+
+this is from app/api/tenants/tenantContext.ts
+```
+export type Tenant = {
+  name: string; //should be unique
+  dbName: string; //should be unique
+  indexName: string; //should be unique
+  uploadedDocuments: string;
+  attachments: string;
+  customUploads: string;
+  temporalFiles: string;
+};
+
+```
+
+uwazi will use this config for requests that provide a Tenant header corresponding with the tenant name, for requests that do not provide one uwazi will use the defaultTenant from app/api/config.ts.
```

Comment:
@daneryl Please, check the capitalization :)

Reply:
done